lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -31,10 +31,10 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
 
       [
         %ElevatorClosures{
-          id: elevator_id,
           in_station_closures: Enum.map(in_station_alerts, &alert_to_elevator_closure/1),
           other_stations_with_closures:
-            format_outside_closures(outside_alerts, parent_station_map, routes_map)
+            format_outside_closures(outside_alerts, parent_station_map, routes_map),
+          elevator_config: config
```

Comment:
`app_params` might be a better name for this — `config` is a little imprecise and has a lot of overlap with the general notion of a screen's whole "config".